test_helpers.py

```
@@ -138,6 +138,17 @@ def ssrf():
     except Exception as e:
         return f"<pre>Failed to fetch URL: {e}</pre>"
 
+# --- SSRF (Server-Side Request Forgery) ---
+@app.route('/ssrf')
+def ssrf():
```

Ruff (0.12.2)

Redefinition of unused `ssrf` from line 132

(F811)